Doc/library/typing.rst

```diff
@@ -2804,6 +2804,47 @@ Introspection helpers
 
    .. versionadded:: 3.8
 
+.. function:: get_orig_bases(tp, /)
```

Comment:
Considering that `__orig_bases__` is set by `__build_class__` and isn't typing-specific, it may make sense to put this in `types` instead. It's closely related to `types.resolve_bases`.